civictechprojects/static/css/partials/_Global.scss

```diff
@@ -18,6 +18,10 @@ body {
 a.btn-theme:hover, a[type="button"].btn-theme:hover {
   text-decoration: none;
 }
+.btn {
```

Comment:
I would add the btn-theme class to the specific buttons, rather than altering a style that's used by just about every button on the site.